jupyter_client/blocking/client.py

```diff
@@ -160,6 +160,9 @@ def _recv_reply(self, msg_id, timeout=None):
     inspect = reqrep(KernelClient.inspect)
     kernel_info = reqrep(KernelClient.kernel_info)
     comm_info = reqrep(KernelClient.comm_info)
+    comm_open = reqrep(KernelClient.comm_open)
+    comm_message = reqrep(KernelClient.comm_message)
+    comm_close = reqrep(KernelClient.comm_close)
```

Comment:
These probably shouldn't be wrapped in `reqrep`, because they don't (in general) have a reply to wait for.